lib/sage.ex

```diff
@@ -382,6 +389,10 @@ defmodule Sage do
 
     * `:timeout` - the time in milliseconds to wait for the transaction to finish, \
     `:infinity` will wait indefinitely (default: 5000);
+    * `:supervisor` - the name of a supervisor that the task will be spawned under. Defaults to the\
+    `Sage.AsyncTransactionSupervisor` started by Sage (meaning it will exist under Sage's supervision\
+    tree). It might be a good idea to pass your own in to avoid race conditions on shutdown if your\
+    step interacts with a process in your own app's supervision tree.
```

Comment:
```suggestion
    * `:supervisor` - the name of a `Task.Supervisor` that the task will be spawned under. Defaults to the\
    `Sage.AsyncTransactionSupervisor` started by Sage (meaning it will exist under Sage's supervision\
    tree). It might be a good idea to pass your own in to avoid race conditions on shutdown if your\
    step interacts with a process in your own app's supervision tree.
```